transaction-service/src/main/java/pl/adamboguszewski/transaction/service/application/TransactionInfo.java

```diff
@@ -4,34 +4,42 @@
 import pl.adamboguszewski.transaction.service.service.api.transaction.CreateTransactionRequest;
 
 import java.time.LocalDateTime;
-import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Value
 public class TransactionInfo {
 
     LocalDateTime transactionDateTime;
-
     String checkoutId;
 
-    List<PaymentInformation> paymentInformations;
+    List<PaymentInformation> payments;
 
-    private TransactionInfo(LocalDateTime dateTime, String checkoutId) {
-        this.transactionDateTime = dateTime;
+    private TransactionInfo(LocalDateTime transactionDateTime, String checkoutId, List<PaymentInformation> payments) {
+        this.transactionDateTime = transactionDateTime;
         this.checkoutId = checkoutId;
-        this.paymentInformations = new ArrayList<>();
+        this.payments = payments;
     }
 
-    private void createPaymentsFromRequest(List<CreateTransactionRequest.TransactionInfo.TransactionPayment> requests) {
-        for(CreateTransactionRequest.TransactionInfo.TransactionPayment paymentRequest
-                : requests) {
-            this.paymentInformations.add(PaymentInformation.fromRequest(paymentRequest));
-        }
-    }
+//    // immutable object bruh..
+//    private void createPaymentsFromRequest(List<CreateTransactionRequest.TransactionInfo.TransactionPayment> requests) {
+//        for(CreateTransactionRequest.TransactionInfo.TransactionPayment paymentRequest
+//                : requests) {
+//            this.payments.add(PaymentInformation.fromRequest(paymentRequest));
+//        }
+//    }
```

Comment:
Ugly commenting style (/* */ ftw). Message understood.

Reply:
try this shortcut first
ctrl + /